staging/src/github.com/kubeedge/beehive/pkg/core/model/message.go

```diff
@@ -254,7 +254,7 @@ func (msg *Message) NewRespByMessage(message *Message, content interface{}) *Mes
 
 // NewErrorMessage returns a new error message by a message received
 func NewErrorMessage(message *Message, errContent string) *Message {
-	return NewMessage(message.Header.ParentID).
```

Comment:
In NewMessage, the parameter is `ParentID` :)

Reply:
> In NewMessage, the parameter is `ParentID` :)

Here, the `feedbackError` function should return the error to the request from `Sendsyn`, but now the `NewError Message` function uses the ParentID of the requested message as the ParentID of the ErrorMessage. Because the ParentID of the sent message is empty, the ID of the Error Message will also be empty, which will cause the message to be discarded and there will be no receiving channel.

Reply:
Sounds good.

Reply:
Sounds good.